Main.cs

```diff
@@ -1,93 +1,211 @@
-       const int fuerzaGuerrero = 10;
-       const int fuerzaArquero = 5;
+        // Constantes de fuerza de cada unidad
+        const int fuerzaAldeana = 0;
+        const int fuerzaGuerrera = 10;
+        const int fuerzaArquera = 5;
+        // Constante de salud/vida
+        const int vida = 20;
+
+        // Antes de la primera elección de turno
+        int equipoAtacante = -1;
 
+        // Variable global de fin de juego (por implementar)
+            // Nota de Sonia:
+                // No se puede poner pública
+        bool gameOver = false;
 
-        
+        // Entero en DEV para contar las vueltas de ejecución
+        // mientras gameOver == true
+        int update = 0;
 
-        // Toda Unidad tiene el mismo valor entero para su salud/vida
-        int vida = 20;
+        // Random() es un método que se debe crear para cada uso
+        // Random aleatorio = new Random();
 
         // Creando una lista vacía para el equipo Azul
         List<Personaje> EquipoAzul = new List<Personaje>();
+        List<Personaje> EquipoRojo = new List<Personaje>();
+
+        // Agregando 4 unidades al equipo azul
+        EquipoAzul.Add(new Aldeano(vida, fuerzaAldeana));
+        EquipoAzul.Add(new Aldeano(vida, fuerzaAldeana));
+        EquipoAzul.Add(new Arquero(vida, fuerzaArquera));
+        EquipoAzul.Add(new Guerrero(vida, fuerzaGuerrera));
+
+        // Agregando 4 unidades al equipo rojo
+        EquipoRojo.Add(new Aldeano(vida, fuerzaAldeana));
+        EquipoRojo.Add(new Aldeano(vida, fuerzaAldeana));
+        EquipoRojo.Add(new Arquero(vida, fuerzaArquera));
+        EquipoRojo.Add(new Guerrero(vida, fuerzaGuerrera));
+
+        while ( ! gameOver ) {
+            // Establecer turnos de ataque
+            // Se inicia con una moneda al aire
+
+            equipoAtacante = SetTurno( equipoAtacante );
+
+            // (DEV) Mostrando las vueltas de ejecución
+            update++;
+            Console.WriteLine( $"Class Program: Ejecutando Main() num {update}. Turno para: {equipoAtacante}.\n" );
+
+            // Turno para el equipo azul == 0
+            if ( equipoAtacante == 0 ) {
+
+                gameOver = LanzarAtaque( EquipoAzul, EquipoRojo );
 
-        // Probando una instancia de la clase Personaje
-        Arquero pepe = new Arquero (
-            vida, 
-            fuerzaArquero 
-        );
+                // Turno para el equipo rojo == 1
+            } else {
 
-        // Agregando un Personaje a la lista del equipo Azul
-        EquipoAzul.Add( pepe );
+                gameOver = LanzarAtaque( EquipoRojo, EquipoAzul );
+            }
+        }
 
-        EquipoAzul[0].Info();
+        // El método Length() es para arrays. Count() para listas
+        if ( EquipoAzul.Count > 0 ) {
+            Console.WriteLine( "Ganador: EQUIPO AZUL." );
+        } else {
+            Console.WriteLine( "Ganador: EQUIPO ROJO." );
+        }
     }
-}
 
+    // Solucionar error de compilación
```

Comment:
**Duda:** Este comentario está aquí intencionado?

Reply:
Sip.
Soluciona error anterio:
error CS0120: An object reference is required for the non-static field, method, or property 'Program.LanzarAtaque(List<Personaje>, List<Personaje>)'

Se cambia el método LanzarAtaque() a static LanzarAtaque()